js/src/dashboard/summary-section/paid-features/index.scss

```diff
@@ -0,0 +1,40 @@
+.gla-paid-features {
+	@media (max-width: $break-small) {
+		&__content {
+			flex-direction: column;
+			gap: $grid-unit-20;
+		}
+	}
+
+	&__feature-list {
+		display: flex;
+		flex-direction: column;
+		gap: $grid-unit-20;
+		line-height: $gla-line-height-smaller;
+		color: $gray-800;
+		text-align: left;
+
+		.gridicon {
+			fill: #5ec862;
+		}
+	}
+
+	.gla-free-ad-credit-claim {
+		text-align: left;
+		background-color: $white;
+		display: flex;
+		align-items: center;
+		gap: calc(var(--main-gap) / 3 * 2);
+		padding: $grid-unit-20 $grid-unit-15;
+		color: $gray-900;
+		font-size: $gla-font-base;
```

Comment:
This font size can be omitted as the font size inherited from the parent is the same.